src/app/shared/formly/dev/testing/formly-testing.module.ts

```diff
@@ -15,6 +15,7 @@ class CheckboxFieldComponent extends FieldType {}
 @Component({
   template: `FieldsetFieldComponent:
     <div *ngFor="let f of field.fieldGroup">
+      {{ f.key }}
```

Comment:
Useful, we should consider adding the key to other fields too so it's consistent.